src/utils/validators.js

```diff
@@ -50,19 +54,7 @@ export const validatePinWithConfirmation = (pin: string, confirmationPin: string
 };
 
 /* eslint-disable i18next/no-literal-string */
-const supportedDomains = [
-  'eth',
-  'crypto',
-  'zil',
-  'nft',
-  'x',
-  'wallet',
-  'bitcoin',
-  'dao',
-  '888',
-  'coin',
-  'blockchain',
-];
+const supportedDomains = ['eth', 'crypto', 'zil', 'nft', 'x', 'wallet', 'bitcoin', 'dao', '888', 'coin', 'blockchain'];
```

Comment:
This is now redundant because we are using the Name Resolution Service to determine what is right or wrong, and new crypto domain suffixes are being added all the time. This check an be removed.